packages/rs-dpp/src/reduced_platform_state/v0/mod.rs

```diff
@@ -0,0 +1,25 @@
+use crate::block::block_info::BlockInfo;
+use crate::block::extended_block_info::ExtendedBlockInfo;
+use crate::fee::default_costs::EpochIndexFeeVersionsForStorage;
+use crate::util::deserializer::ProtocolVersion;
+use bincode::{Decode, Encode};
+use platform_value::Bytes32;
+
+/// Reduced Platform State for Saving V0
+#[derive(Clone, Debug, Encode, Decode)]
+pub struct ReducedPlatformStateForSavingV0 {
```

Comment:
So you want to duplicate these data in both PlatfromState and ReducedPlatform state? and serialize and store it twice a block?

Reply:
As this complicates the logic, I propose to make a separate PR for this, since it is only an optimisation/refactoring.